Libraries/Components/MapView/MapView.js

```diff
@@ -14,26 +14,15 @@
 var EdgeInsetsPropType = require('EdgeInsetsPropType');
 var Image = require('Image');
 var NativeMethodsMixin = require('NativeMethodsMixin');
-var Platform = require('Platform');
 var RCTMapConstants = require('NativeModules').UIManager.RCTMap.Constants;
 var React = require('React');
-var ReactNativeViewAttributes = require('ReactNativeViewAttributes');
 var View = require('View');
 
-var deepDiffer = require('deepDiffer');
-var insetsDiffer = require('insetsDiffer');
-var merge = require('merge');
 var processColor = require('processColor');
 var resolveAssetSource = require('resolveAssetSource');
 var requireNativeComponent = require('requireNativeComponent');
 
 type Event = Object;
-type MapRegion = {
-  latitude: number;
-  longitude: number;
-  latitudeDelta: number;
-  longitudeDelta: number;
-};
```

Comment:
This type may be used internally in FB's codebase (@fkgozali, mind checking? :)). In either case, it'd be sad to turn the clock back on the flowification of the codebase. Just use the type below somewhere so lint is happy?